sdk/src/rpc/types/request.ts

```diff
@@ -1,387 +1,358 @@
     allocations: AppSessionAllocation[];
 }
+export type CloseAppSessionRPCRequestParams = CloseAppSessionRequestParams; // for backward compatibility
 
 /**
  * Represents the request structure for the 'close_app_session' RPC method.
  */
-export interface CloseAppSessionRPCRequest extends GenericRPCMessage {
+export interface CloseAppSessionRequest extends GenericRPCMessage {
     method: RPCMethod.CloseAppSession;
-    params: CloseAppSessionRPCRequestParams;
+    params: [CloseAppSessionRequestParams];
 }
 
 /**
  * Represents the request parameters for the 'get_app_definition' RPC method.
  */
-export interface GetAppDefinitionRPCRequestParams {
+export interface GetAppDefinitionRequestParams {
     /** The unique identifier of the application session to retrieve. */
     app_session_id: Hex;
 }
+export type GetAppDefinitionRPCRequestParams = GetAppDefinitionRequestParams; // for backward compatibility
 
 /**
  * Represents the request structure for the 'get_app_definition' RPC method.
  */
-export interface GetAppDefinitionRPCRequest extends GenericRPCMessage {
+export interface GetAppDefinitionRequest extends GenericRPCMessage {
     method: RPCMethod.GetAppDefinition;
-    params: GetAppDefinitionRPCRequestParams;
+    params: [GetAppDefinitionRequestParams];
 }
 
 /**
  * Represents the request parameters for the 'get_app_sessions' RPC method.
  */
-export interface GetAppSessionsRPCRequestParams {
-    /** Optional participant address to filter application sessions. If not provided, returns all sessions. */
-    participant?: Address;
-    /** Optional status to filter application sessions (e.g., "open", "closed"). If not provided, returns sessions of all statuses. */
-    status?: RPCChannelStatus;
+export interface GetAppSessionsRequestParams {
+    /** The participant address to filter application sessions. */
+    participant: Address;
+    /** The status to filter application sessions (e.g., "open", "closed"). */
+    status: RPCChannelStatus;
 }
+export type GetAppSessionsRPCRequestParams = GetAppSessionsRequestParams; // for backward compatibility
 
 /**
  * Represents the request structure for the 'get_app_sessions' RPC method.
  */
-export interface GetAppSessionsRPCRequest extends GenericRPCMessage {
+export interface GetAppSessionsRequest extends GenericRPCMessage {
     method: RPCMethod.GetAppSessions;
-    params: GetAppSessionsRPCRequestParams;
+    params: [GetAppSessionsRequestParams];
 }
 
 /**
  * Represents the request parameters for the 'resize_channel' RPC method.
  */
-export interface ResizeChannelRPCRequestParams {
+export type ResizeChannelRequestParams = {
     /** The unique identifier of the channel to resize. */
     channel_id: Hex;
-    /** Optional amount to resize the channel by (can be positive or negative). Must be provided if allocate_amount is not. */
+    /** Amount to resize the channel by (can be positive or negative). Required if allocate_amount is not provided. */
     resize_amount?: bigint;
```

Comment:
These `...Params` types are passing from business logic to sdk's `create<insert RPC method>Message` method, meaning that it will be used in higher domain, where snake_case is  not acceptable. Use camelCase here and construct message for sending to BE inside sdk methods.

Also, this line showcase another problem: standart JSON package doesn't know how to work with bigint, so before calling `JSON.stringify` make sure that structure inside doesn't have bigint (use string instead)